testdata/ansible/memcached-operator/bundle/tests/scorecard/config.yaml

```diff
@@ -12,38 +12,59 @@ stages:
     labels:
       suite: basic
       test: basic-check-spec-test
+    storage:
```

Comment:
The storage spec has `omitempty` on it, so these shouldn't be showing up here.

Reply:
odd... It looks like omitempty is missing from the `spec` field, could that be causing this? sanity check won't pass with these removed, might need to followup to add that in api, would this be blocking?

Reply:
Yeah that looks correct. Not a blocker, just needs a follow-up.